README.rst

```diff
@@ -1,15 +1,7 @@
 ====================================================
-EnCore - A collection of core-level utility modules
+Encore - A collection of core-level utility modules
```

Comment:
The naming isn't entirely consistent everywhere, but "Encore" seems to be more common in the codebase than "EnCore". We should pick one or the other. If anyone wants to argue for EnCore everywhere instead of Encore, that's fine with me.

Reply:
From internal discussion: it should be Encore, not EnCore.